package.json

```diff
@@ -32,6 +32,7 @@
   "devDependencies": {
     "chai": "^3.2.0",
     "coveralls": "^2.11.2",
+    "graceful-fs": "4.1.4",
```

Comment:
With OSX + node 0.10, the error happened every time on the before hook (https://github.com/paulmillr/chokidar/blob/master/test.js#L66-L68). I think this limit on the number of file descriptors may also be observed on other node versions if more tests are added to the suite.